### PR TITLE
INSP: properly proofread single sentence in sequential Rust comments

### LINKED ISSUE #7024: Grazie doesn't take into account sequential line comments as continuous text in Rust files
## Environment

* **IntelliJ Rust plugin version:** 0.3.145.3754-211-nightly
* **Rust toolchain version:** 1.52.0-nightly (3a5d45f68 2021-03-09) x86_64-apple-darwin
* **IDE name and version:** IntelliJ IDEA 2021.1 Beta Ultimate Edition (IU-211.6693.14)
* **Operating system:** macOS 10.15.6
* **Macro expansion engine:** new
* **Name resolution engine:** new

## Problem description
image

Similar to https://youtrack.jetbrains.com/issue/IDEA-230091
## Steps to reproduce
Enable `Rust` option in `Preferences | Editor | Proofreading | Grammar` settings and write the following code:

```rust
fn main() {
    // Path to directory where someone writes
    // and from where someone reads <----- Here Grazie shows a warning, suggesting that it is incorrect to start a sentence with "and"
    println!("Hello!")
}
```

### PR DESCRIPTION
**Pull request by Undin**
Also, don't use yet another deprecated API

Fixes #7024

changelog: Fix false-positive proofreading warnings for sentences written in several sequential Rust comments. Note, you can enable/disable proofreading in Rust files in `Preferences | Editor | Proofreading | Grammar` settings

### CODE REVIEW

File: grazie/src/test/kotlin/org/rust/grazie/RsGrammarCheckingTest.kt [hunk 1]
```diff
@@ -53,6 +53,18 @@ class RsGrammarCheckingTest : RsInspectionsTestBase(GrazieInspection::class) {
         }
     """, checkInComments = true)
 
+    // https://github.com/intellij-rust/intellij-rust/issues/7024
+    fun `test check single sentence in sequential comments`() = doTest("""
+        fn main() {
+            // Path to directory where someone <TYPO>write</TYPO>
+            // and from where someone reads
```
Comment (vlad20012) [inline grazie/src/test/kotlin/org/rust/grazie/RsGrammarCheckingTest.kt:60]: Let's add a test with an empty line between comment. In this case, a `typo` should be emitted

**Review by vlad20012 — APPROVED** [review]

**Comment by Undin** [comment]
bors r=vlad20012

**Comment by Undin** [comment]
bors retry
